prql-net/PrqlCompiler/PrqlCompiler.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>Prql.Compiler</PackageId>
+    <PackageVersion>0.1.0</PackageVersion>
```

Comment:
Should we align this with our standard version?

We can use the replacements in `Cargo.toml` so it updates on every release.

Reply:
I don't know. Maybe. We could that. This is a early pre-release though, I would really like the prql-lib to be updated to expose the `compile` function if possible so I don't have to use the `to_sql` function which does not accept PRQL compiler options.

Reply:
OK, if we don't do that now, can we add a TODO saying that in the code?

Reply:
To say something like `# TODO: this is currently at 0.1.0 because we're waiting to update prql-lib for the latest API. When we've done that, we can match the version here with the broader PRQL version`.

Generally we do this so others know what state the project is in, and what the next steps are to make progress. Does that make sense? (Not trying to slow you down, tell me if this is a burden and we can skip it...)

Reply:
Alright, I added a TODO to the README.

Yeah, that is fine.